atc/event/events.go

```diff
@@ -176,6 +176,14 @@ type FinishPut struct {
 func (FinishPut) EventType() atc.EventType  { return EventTypeFinishPut }
 func (FinishPut) Version() atc.EventVersion { return "5.1" }
 
+type SetPipelineChanged struct {
+	Origin  Origin `json:"origin"`
+	Changed bool   `json:"changed"`
+}
+
+func (SetPipelineChanged) EventType() atc.EventType  { return EventTypeSetPipelineChanged }
+func (SetPipelineChanged) Version() atc.EventVersion { return "1.0" }
+
```

Comment:
More of a note for the future than something to bundle in here:

This new event type could also be rendered in `fly watch` by adding an extra case in `fly/eventstream/render.go`. That could be a quick follow-up for someone to add, or maybe you just want to slipstream that here. No strong feeling from me on that but I do think it would be nice to have rough parity between the textual and web views.